pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -120,9 +120,10 @@ function getSpaceAvailable(storagePools, connectionName) {
 function validateParams(vmParams) {
     const validationFailed = {};
 
-    if (isEmpty(vmParams.vmName.trim())) {
+    if (isEmpty(vmParams.vmName.trim()))
         validationFailed.vmName = _("Name should not be empty");
-    }
+    else if (/\s/.test(vmParams.vmName))
```

Comment:
There is #12893 which is related to this.
I think it would be better if we would just remove spaces from the name in order to use it as disk name. (`name.split(" ").join("")` would be enough)

Reply:
If someone types spaces, they're probably doing so for a reason. It would be better to use spaces (if we can, please — I don't understand the random hate for spaces sometimes) or at least convert them to underscores.

Reply:
Agreed, converting spaces to - or _ is better than just dropping them.

Reply:
Fixed with converting spaces to underscores.